packages/module-sandbox/src/app/ModuleSelector.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Box, MenuItem, TextField } from '@material-ui/core'
+
+export interface ModuleSelectorProps extends React.HTMLAttributes<HTMLElement> {
+  moduleId: string | undefined
+  setModuleId: (moduleId: string | undefined) => void
+}
+
+export const ModuleSelector = ({ moduleId, setModuleId, ...rest }: ModuleSelectorProps): JSX.Element => {
+  const moduleIds = window.__lumy_dynamicComponentsRegistry?.ids ?? []
+  return (
+    <Box justifyContent="space-around" display="flex" {...rest}>
+      <TextField
+        id="outlined-select-currency"
+        select
+        label="Module"
+        value={moduleId ?? ''}
+        onChange={e => setModuleId(e.target.value as string)}
+        // helperText="Select module"
```

Comment:
helperText unnecessary?

Reply:
I'm not sure. What do you think? :)

Reply:
It seems that you don't need it here. :)